File: ass37/program1.c
/*
1.Write a program which accept string from user count number of words from string  
Input :   “Marvellous Multi OS” 
Output :   3 
Input :   “          Marvellous Multi            OS       Pune” 
Output :   4
*/
#include<stdio.h>
int CountWord(char *str)
{
    int iCnt = 0;
    if(str==NULL)
    {
        return -1;
    }
    
    while(*str!='\0')
    {
    if((*str>='A')&&(*str<='Z'))
    {
        *str = *str + 32;
        iCnt++;
    }
    else
    {
        *str = *str - 32;
    }
    str++;
    }
    
        printf("%d\n",iCnt);
}
int main()
{
 //   int bRet ;
    char arr[50]="Marvellous Multi OS";
    CountWord(arr);
    
    return 0;
}

File: ass55/Program1.java
/*
1. Write java program which accept N numbers from user and accept one another number as NO , check whether NO is present or not. 
Input :  N :   6 
  NO:  66 
  Elements : 85 66 3 66 93 88  
Output : TRUE 
Input :  N :   6 
  NO:  12 
  Elements : 85 11 3 15 11 111  
Output : FALS
*/
import java.lang.*;
import java.util.*;
class ArrayDemo
{
    boolean Even(int arr[],int value)
    {
        int iSum1 = 0;
        int itemp = value;
        for(int i=0; i<arr.length; i++)
        {
            if(arr[i]==value)
            {
               break;
            }   
        }
        if(arr[i]==itemp )
        {
            return true;
        }
        else
        {
            return false;
        }
    }
}
class Program4
{
    public static void main(String arg[])
    {
        boolean bret = false;
        Scanner sobj = new Scanner(System.in);
        System.out.println("Enter the Element");
        int size = sobj.nextInt();
        int value = sobj.nextInt();
        int arr[] = new int[size];
        System.out.println("Enter the Array elements");
        for(int i=0; i<arr.length; i++)
        {
            arr[i] = sobj.nextInt();
        } 
        ArrayDemo dobj = new ArrayDemo();
        bret = dobj.Even(arr,value);
        if(bret == true)
        {
            System.out.println("NO is Present"); 
        }
        else
        {
            System.out.println("NO is not Present"); 
        }
    }
}
